<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutDialog</class>
 <widget class="QDialog" name="AboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About Masonry</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="1" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="4">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Masonry</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLabel" name="organizationLabel">
     <property name="text">
      <string>%1 (http://%2)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QLabel" name="versionLabel">
     <property name="text">
      <string>Version %1</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Copyright (c) 2009 ifak - Institut f√ºr Automation und Kommunikation e.V.</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>A graphical tool to visualize and compute Mason-graphs.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
